typescript/sdk/src/core/environments/test.ts

```diff
@@ -1,113 +1,137 @@
 export const addresses = {
   alfajores: {
     upgradeBeaconController: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
-    xAppConnectionManager: '0x610178dA211FEF7D417bC0e6FeD39F05609AD788',
-    validatorManager: '0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512',
-    interchainGasPaymaster: '0x8A791620dd6260079BF849Dc5567aDC3F2FdC318',
+    xAppConnectionManager: '0x0165878A594ca255338adfa4d48449f69242Eb8F',
+    interchainGasPaymaster: '0x5FC8d32690cc91D4c39d9d3abcBD16989F875707',
+    outboxMultisigValidatorManager:
+      '0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512',
+    inboxMultisigValidatorManagers: {
```

Comment:
I considered merging this object with the existing `inboxes` object (& the same idea for outbox), something like:
```
inboxes: {
  kovan: {
    multisigValidatorManager: '0x...',
    inbox: {
      proxy: '0x...',
      implementation: '0x...',
      beacon: '0x...',
    }
  }
}
```

But wanted to hear some thoughts before going too far down that route. Anyone have opinions?

Reply:
I like it

Reply:
thinking about this more, I think it may muddle the distinction between an inbox contract and a validator manager contract, which is made more awkward by all other contracts being very distinct in the config. I don't mean just visually/logically within the config file but how the shape would affect `CoreContractAddresses`. So I think I'll leave as is